checkstyle.xml

```diff
@@ -53,7 +53,12 @@
         <module name="TypeName"/>
         <module name="IllegalImport"/> <!-- defaults to sun.* packages -->
         <module name="IllegalImport">
-            <property name="illegalPkgs" value="clover, junit.framework, org.apache.commons.lang, org.slf4j, com.google.common.collect"/>
+            <property name="illegalPkgs" value="clover,
+                                                com.google.common.collect,
+                                                junit.framework,
+                                                org.apache.commons.lang,
+                                                org.apache.log4j,
```

Comment:
I added `org.apache.log4j`

The rest is just alphabetizing and breaking up on lines. Happy to revert that junk if you prefer.